tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts

```diff
@@ -142,38 +142,6 @@ export class FromPixelsProgram implements WebGPUProgram {
     return this.inputTexture;
   }
 
-  generateEncoder(device: GPUDevice, output: GPUBuffer): GPUCommandEncoder {
-    const bindGroup = device.createBindGroup({
-      layout: this.bindGroupLayout,
-      entries: [
-        {
-          binding: 0,
-          resource: {
-            buffer: output,
-          }
-        },
-        {
-          binding: 1,
-          resource: this.inputTexture.createView(),
-        },
-        {
-          binding: 2,
-          resource: {
-            buffer: this.uniform,
-          }
-        }
-      ],
-    });
-
-    const commandEncoder = device.createCommandEncoder({});
```

Comment:
I have to move `generateEncoder` to `backend_webgpu.ts` due to encoder needed to be created if backend.currentCommandEncoder is null. However, if we pass currentCommandEncoder to this function, it will create a new encoder instead of the original currentCommandEncoder. And it also doesn't make sense to bring backend to this file. So I move it to backend_webgpu.ts.